portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -45,23 +56,127 @@ public void tearDown() {
     helper.tearDown();
   }
 
+  @Test 
+  public void testPOSTSingleComment() throws Exception {
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+    HttpServletRequest postRequest = mock(HttpServletRequest.class);
+    HttpServletResponse postResponse = mock(HttpServletResponse.class);
+
+    Assert.assertEquals(0, ds.prepare(new Query(COMMENT_TABLE_NAME)).countEntities(withLimit(10)));
```

Comment:
nit - can you switch all of these Assert invocations over to their truth evuivalents?

Reply:
done!